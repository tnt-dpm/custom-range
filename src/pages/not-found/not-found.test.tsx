import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import NotFound from '.';

describe('NotFound', () => {
  it('should render snapshot', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});