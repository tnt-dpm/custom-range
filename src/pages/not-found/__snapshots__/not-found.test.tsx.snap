// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="not-found"
  >
    <div>
      <h1
        class="not-found__title"
      >
        404
      </h1>
      <div
        class="not-found__card"
      >
        <p
          class="not-found__desc"
        >
          No hemos encontrado esta p√°gina.
        </p>
        <p
          class="not-found__hint"
        >
          Usa la barra lateral para ir a un ejercicio.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
